### PR TITLE
Include value in metrics for unspecified value

### LINKED ISSUE #181: [Proposal] detailed metric mode, for value in descriptor KEY_VALUE pair, if value not explicitly defined/matched in config
In order to gain greater observability of the operations of the rate limit service, a "detailed" metric mode that exposes metrics on the descriptor KEY_VALUE pairs in all cases, not only when the value is explicitly configured in the config.
These _values_ would allow for substantially more granular alerts / downstream reporting based on metrics emitted by the process when a descriptor key is matched, but its value is not.

While there's an understandable the concern over metric cardinality when including the values in the metrics, it is often the case that the possible values are well understood, and limited such that including them would pose little concern.

### PR DESCRIPTION
**Pull request by jespersoderlund**
Fixes #181 by adding a configuration key to a ratelimit descriptor to allow it to include the value in a statistics value.

### CODE REVIEW

**Comment by jespersoderlund** [comment]
Renamed the config-parameter to "detailed_metric" instead of the crazy-long config parameter I used first

File: README.md [hunk 1]
```diff
@@ -490,6 +499,43 @@ descriptors:
           unit: second
 ```
 
+#### Example 8
+
+In this example there of where the extended metrics for a key where there will be detailed metrics even for the unknown values.
```
Comment (mattklein123) [inline README.md:504]: Typo? Can't parse.

**Review by mattklein123 — CHANGES_REQUESTED** [review]
LGTM with small comment nit, thanks.

File: README.md (hunk 1 above)
Comment (jespersoderlund) [reply README.md]: Yeah, not very clear, fixed

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Comment by marinasalmen** [comment]
Hey @mattklein123 I think something went wrong with the image build and push.
Could you maybe take a look at it? Looking forward for this feature! 🙌

**Comment by mattklein123** [comment]
This should be fixed. There was a credential issue.

**Pull request merged**